ceph_deploy/hosts/remotes.py

```diff
@@ -36,10 +36,12 @@ def platform_information(_linux_distribution=None):
                 codename = minor
             else:
                 codename = major
-    if not codename and 'oracle' in distro.lower(): # this could be an empty string in Oracle linux
+    if not codename and 'oracle' in distro.lower():  # this could be an empty string in Oracle linux
```

Comment:
accidentally change?, same for the next line below?

Reply:
yes, auto pep8 added an additional space to the comment